SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLGetVehicleDataSpec.m

```diff
@@ -110,7 +112,9 @@
                                                                    SDLRPCParameterNameSteeringWheelAngle:@NO,
                                                                    SDLRPCParameterNameTirePressure:@YES,
                                                                    SDLRPCParameterNameTurnSignal:@NO,
-                                                                   SDLRPCParameterNameWiperStatus:@YES},
+                                                                   SDLRPCParameterNameWiperStatus:@YES,
+                                                                   @"stabilityControlsStatus":@YES
```

Comment:
`SDLRPCParameterNameStabilityControlsStatus `

Reply:
If I replace the strings with constants declared in another file how can I be sure the constants are correct?
In such case the test will test nothing will it not?